inc/body/tna-header.inc

```diff
@@ -6,7 +6,7 @@
                 <button aria-label="Toggle menu" id="mega-menu-mobile"></button>
                 <a href="http://www.nationalarchives.gov.uk" title="Go to The National Archives homepage"
                    class="visible-lg visible-md visible-sm">
-                    <img src="./img/logo-white.png" alt="The National Archives"
+                    <img src="../img/logo-white.png" alt="The National Archives"
```

Comment:
This should be using the SITE_URL function, not ../ otherwise the images do not load on the landing page